partner_communication_switzerland/data/sponsorship_communications_cron.xml

```diff
@@ -40,5 +31,87 @@
             <field name="code">model.generate_tax_receipts()</field>
             <field name="active" eval="False"/>
         </record>
+
+        <!-- Sponsorship anniversaries (1, 3, 5, 10 and 15 years) -->
+        <record id="sponsorship_anniversary_1_communication" model="base.automation">
+            <field name="name">Send sponsorship anniversary 1 year communication</field>
+            <field name="model_id" ref="model_recurring_contract"/>
+            <field name="config_id" ref="planned_anniversary_1"/>
+            <field name="state">communication</field>
+            <field name="partner_field">correspondent_id</field>
+            <field name="filter_domain">[("start_date", ">=", Datetime.now() - relativedelta(years=1)), ("state", "=", "active"), ("child_id", "!=", False")]</field>
```

Comment:
I think that I discovered the the field `last_run` is used to check if the rule was already checked, and we can use it to avoid sending the communications that were already sent in the past with CRON like this:
```suggestion
            <field name="filter_domain">[("state", "=", "active"), ("child_id", "!=", False")]</field>
            <field name="last_run" eval="Datetime.now()"/>
```
I think if we force to say that the last run was now, it will only take the new records starting from now on. Maybe you can test this and if it works change it everywhere.